README.md

```diff
@@ -41,9 +41,18 @@ Give a ⭐ if you like this project!
 
 This project is [MIT](./MIT.md) licensed.
```

Comment:
Great job @degisew ✋

- [ ] Kindly use the right readme file for your projects. To get started, you may use this [readme template](https://github.com/microverseinc/readme-template) provided by Microverse. You may also fix the broken MIT license link to make your readme file more professional.

Reply:
Hello @1GORDON 
Thanks for your comment. I have one question for you. How can I fix the broken MIT license on README?

Reply:
This change has been done well